ajax/libs/jquery.flipster/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "jquery.flipster",
+  "version": "1.1.3",
+  "homepage": "http://brokensquare.com/Code/jquery-flipster/demo/",
+  "description": "Responsive coverfpsterlow-style carousel plugin for jQuery",
+  "filename": "jquery.flipster.min.js",
+  "author": "Adrien Delessert <a@drien.com>",
+  "keywords": [
+    "coverflow",
+    "jquery",
+    "carousel",
+    "flipster",
+    "jquery-plugin",
+    "ecosystem:jquery"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com:drien/jquery-flipster.git"
```

Comment:
This URL can't access.
Let's revise to `git://github.com/drien/jquery-flipster.git`
Or follow the upstream git://github.com:drien/jquery-flipster.git
Thanks.